components/level/game.tsx

```diff
@@ -100,6 +101,31 @@ export default function Game({
     }
   }, [gameState.moveCount, onMove]);
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const SECOND = 1000;
+  const fetchPlayAttempt = useCallback(throttle(30 * SECOND, async () => {
+    if (disableServer || trackingStats === false) {
+      return;
+    }
+
+    const resp = await fetch('/api/play-attempt', {
+      body: JSON.stringify({
+        levelId: level._id,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      method: 'POST',
+    });
+    const response = await resp.json();
+
+    if (resp.status === 412) {
```

Comment:
I'm not sure if we need the 412 response, because there's always a chance that they could break a record. Also it may be interesting to keep track of total play time (even after completion)

Reply:
if the record is broken then actually this would get bypassed because the stat wouldnt return. So hypothetically this would still function. But 412 is probably not the right status code. maybe ill change the status code to 422 instead of 412?

Reply:
Actually re: my messages in discord, I think we don't need to handle this case at all, just continue to track the time like normal after you've completed a level